# Id of the Lama object associated with the descriptor (outside the database).
int32 object_id
string object_type
int32 descriptor_id 
string interface_name
